### PR TITLE
KWoC 2023 report

### PR DESCRIPTION
**Pull request by tejasexpress**

### CODE REVIEW

File: blog/2024-06-12-kwoc-annual-report-2023.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
+---
+slug: kwoc-annual-report-2023
+title: KWoC Annual Report 2023
+tags:
+  - koss
```
Comment (harshkhandeparkar) [inline blog/2024-06-12-kwoc-annual-report-2023.md]: The tags need to be added to the tags.yml file.

File: blog/2024-06-12-kwoc-annual-report-2023.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
+---
+slug: kwoc-annual-report-2023
+title: KWoC Annual Report 2023
```
Comment (harshkhandeparkar) [inline blog/2024-06-12-kwoc-annual-report-2023.md]: Please add author.

File: blog/2024-06-12-kwoc-annual-report-2023.md [hunk 3]
```diff
@@ -0,0 +1,79 @@
+---
```
Comment (harshkhandeparkar) [inline blog/2024-06-12-kwoc-annual-report-2023.md:1]: Can rename it to KWoC 2023 Report

File: blog/2024-06-12-kwoc-annual-report-2023.md [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+# KWoC 2023 
+
+Hello everyone!
+
+The Kharagpur Winter of Code (KWoC) 2023 has come to a close, and what an incredible journey it has been! For those unfamiliar, KWoC is an annual five-week program where students, many of whom are new to open-source software, contribute to coding projects under the guidance of experienced mentors. This program, hosted by the Kharagpur Open Source Society (KOSS) at IIT Kharagpur, is open to students from all universities, creating a diverse and inclusive environment.
+
+![](https://i.imgur.com/cbZnwzj.png)
+## The Motivation
+
+KWoC's main aim is to provide students with one-on-one mentorship as they take their first steps into the world of open source. It helps build bridges between budding developers and experienced mentors. This early guidance is especially beneficial for freshmen and sophomores, helping them build confidence and skills that will be invaluable for larger programs like Google Summer of Code (GSoC).
```
Comment (harshkhandeparkar) [inline blog/2024-06-12-kwoc-annual-report-2023.md]: I don't think we should write one on one mentorship - we don't really expect that from mentors.

**Review by harshkhandeparkar — CHANGES_REQUESTED** [review]

File: blog/2024-06-12-kwoc-annual-report-2023.md (hunk 2 above)
Comment (tejasexpress) [reply blog/2024-06-12-kwoc-annual-report-2023.md]: added

File: blog/2024-06-12-kwoc-annual-report-2023.md (hunk 3 above)
Comment (tejasexpress) [reply blog/2024-06-12-kwoc-annual-report-2023.md:1]: changed

File: blog/2024-06-12-kwoc-annual-report-2023.md (hunk 4 above)
Comment (tejasexpress) [reply blog/2024-06-12-kwoc-annual-report-2023.md]: changed

File: blog/2024-06-12-kwoc-annual-report-2023.md [hunk 5]
```diff
@@ -0,0 +1,80 @@
+---
+slug: kwoc-annual-report-2023
+title: KWoC 2023 Report
+authors: tejasexpress
+tags:
+  - KOSS
```
Comment (harshkhandeparkar) [inline blog/2024-06-12-kwoc-annual-report-2023.md]: I don't think there is any need to have a tag named "KOSS" but a KWoC tag would be good, we will probably make more KWoC blogs in the future.

Comment (harshkhandeparkar) [reply blog/2024-06-12-kwoc-annual-report-2023.md]: Ok the KOSS tag exists so let's keep it but add a KWoC tag.

Comment (harshkhandeparkar) [reply blog/2024-06-12-kwoc-annual-report-2023.md]: While you're at it, add the tag to the other KWoC blog too.

File: blog/tags.yml [hunk 6]
```diff
@@ -1,9 +1,14 @@
...
   label: KOSS
   permalink: /koss
-  description: A bunch of things specific to Kharagpur Open Source Society
\ No newline at end of file
+  description: A bunch of things specific to Kharagpur Open Source Society
+
+kwoc:
+  label: KWoC
+  permalink: /kwoc
+  description: bolgs related to Kharagpur Winter of Code
```
Comment (harshkhandeparkar) [inline blog/tags.yml]: `B` in `Bolgs` can be capital.

Comment (tejasexpress) [reply blog/tags.yml]: changd

**Review by harshkhandeparkar — APPROVED** [review]

**Comment by harshkhandeparkar** [comment]
Thanks!

**Pull request merged**